test/parallel/test-httpsServerOptions.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const https = require('https');
+const fs = require('fs');
+
+var options = {
+  key: fs.readFileSync(common.fixturesDir + '/keys/agent1-key.pem', 'ascii'),
+  crt: fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem', 'ascii')
+};
+
+assert.throws(function() {
+  https.createServer(options, function (req, res) {
+    res.send(200);
+  }).listen(common.PORT);
+}, Error);
```

Comment:
This test only tests `cert` case. What about `key` missing case? It would be better if the error messages are checked explicitly.